type.go

```diff
@@ -347,6 +347,89 @@ func (t booleanType) ConvertValue(val Value, typ Type) (Value, error) {
 	}
 }
 
+type int8Type struct{}
+
+func (t int8Type) String() string                           { return "INT8" }
+func (t int8Type) Kind() Kind                               { return Int32 }
+func (t int8Type) Length() int                              { return 8 }
+func (t int8Type) EstimateSize(n int) int                   { return n }
```

Comment:
I'm not sure if this should actually be n * 4 because we use int32Pages as the underlying representation?